src/tldraw/datatype.ts

```diff
@@ -0,0 +1,26 @@
+import { init as tldrawinit } from "automerge-tldraw"
+
+import { PenBox } from "lucide-react"
+
+// When a copy of the document has been made,
+// update the title so it's more clear which one is the copy vs original.
+// (this mechanism needs to be thought out more...)
+export const markCopy = (doc: any) => {
+  console.log("not implemented", doc)
+}
+
+const getTitle = (doc: any) => {
+  return doc.store["document:document"].name || "Drawing"
```

Comment:
let's use the page name instead (until topbar editor is implemented) and set a better page name than Page 1 by default

Reply:
Steve says there will be a UI for this shortly so I'll just leave it be. (Though I will set it to a non-empty value in init)